app/src/main/java/com/eveningoutpost/dexdrip/Models/ReadingData.java

```diff
@@ -17,9 +17,9 @@
     public List<GlucoseData> history;  // Per 15 minutes data.
     public byte[] raw_data;
 
-    static final int ERROR_INFLUENCE = 4; //  The influence of each error
-    static final int PREFERED_AVERAGE = 5; //  Try to use 5 numbers for the average
-    static final int MAX_DISTANCE_FOR_SMOOTHING = 7; //  If points have been removed, use up to 7 numbers for the average.
+    static final private byte ERROR_INFLUENCE = 4; //  The influence of each error
```

Comment:
Sorry to bother again but please move the variable scope to the beginning of the line to be consistent with the other variable definitions.

Reply:
Done